Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -166,9 +177,32 @@ public final class FormAddressItem: FormValueItem<PostalAddress, AddressStyle>,
         case .stateOrProvince:
             observers[.stateOrProvince] = bind(item.publisher, to: self, at: \.value.stateOrProvince)
             publisherObservers[.stateOrProvince] = observe(publisher) { item.value = $0.stateOrProvince ?? "" }
+        case .country:
+            break
+        }
+    }
+    
+    private func updateItems(identifier: String?) {
+        let items = items + [headerItem, countrySelectItem]
+        for item in items {
+            if let item = item as? FormSplitItem {
+                update(item: item.leftItem, identifier: identifier)
+                update(item: item.rightItem, identifier: identifier)
+                continue
+            }
+            update(item: item, identifier: identifier)
         }
     }
     
+    private func update(item: FormItem, identifier: String?) {
+        let className = String(describing: self)
+        let currentIdentifier = item.identifier
+        let newIdentifier = currentIdentifier?.replacingOccurrences(of: className, with: identifier ?? "")
+        item.identifier = newIdentifier
```

Comment:
this algorithm makes an assumption that the currentIdentifier contains the className, wondering if there is a better way 🤔

Reply:
Right now, every item created on `FormFullAddressItem` is using `self` as the scope for the identifier build. I tried passing the identifier as part of the initializer parameters but we ran into a problem. In some scenarios, the address item is created at the parent class (e.g. AbstractPersonalInformationComponent) so we gotta have a way to change it afterwards, that's why I used a property observer on the `identifier` variable.

Since all items have this identifier format: `FormFullAddressItem.xxx`, it was easier to replace `FormFullAddressItem` by the newly assigned identifier of the address item.

If you wanna know more, glad to jump into a call to show you the problem and why I solved it this way.

Reply:
So no way to pre-se identifir?

Reply:
observer on the `identifier`?, that's too complicated and a bit too much, it should be much simpler than this, lets have a call to understand the problem.